std_msgs/Header header
int64 actuator_mode      # actuator_mode reading from 0 to 3
